aspnetcore/fundamentals/static-files.md

```diff
@@ -59,6 +59,12 @@ For a request to access *test.png*, configure the static files middleware as fol
 
 A request to `http://<app>/StaticFiles/test.png` will serve the *test.png* file.
 
+Using `StaticFileOptions()`, you can also set response headers. For example, you can add a `Cache-Control` header, as you see here making the static files served from the *wwwroot* folder publicly cacheable for 10 minutes (600 seconds):
```

Comment:
consider:
`StaticFileOptions()` can set response headers. For example, the code below allows static files to be served from the *wwwroot* folder publicly cacheable for 10 minutes (600 seconds):

or maybe 
`StaticFileOptions()` can also set response headers.